docs/tutorials/python/linear-regression.md

```diff
@@ -11,6 +11,13 @@ import mxnet as mx
 import numpy as np
 ```
 
+## Prerequisites
+1. [MXNet](http://mxnet.io/get_started/install.html), [Graphviz](../prerequisites/install-graphviz.html).
+2. Jupyter Notebook and Graphviz python packages.
```

Comment:
Can we add a link to the Jupyter page and other packages that you sue?
[Jupyter Notebook](http://jupyter.org/)